3515/2/commands/logout.rb

```diff
@@ -0,0 +1,10 @@
+module Logout
+  def logout!(*)
+    if !(User[from['id']]).to_s.empty?
```

Comment:
You had comment for that check and still did't resolve it. This line is unreadable